client/src/schemas/material.js

```diff
@@ -0,0 +1,156 @@
+import { object, string, array, boolean } from 'yup'
+
+export const categories = [
+  'CELL_LINE',
+  'PLASMID',
+  'PROTOCOL',
+  'DATASET',
+  'MODEL_ORGANISM',
+  'PDX',
+  'OTHER'
+]
+
+export const importSources = [
+  'GEO',
+  'SRA',
+  'DBGAP',
+  'DATASET',
+  'PROTOCOLS_IO',
+  'ADDGENE',
+  'JACKSON_LAB',
+  'ATCC',
+  'ZIRC',
+  'OTHER'
+]
+
+const materialCategories = {
+  CELL_LINE: object({
+    cell_line_name: string(),
+    tissue: string(),
+    disease: string(),
+    number_of_availible_cell_lines: string(),
```

Comment:
This shouldn't be an int?

Reply:
I think every category has a field for the number of them, so I'm gonna stop commenting on each one.

Reply:
same comment and above